node/parachain/src/command.rs

```diff
@@ -40,7 +40,7 @@ fn load_spec(
 	para_id: ParaId,
 ) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
 	match id {
-		"" => Ok(Box::new(chain_spec::get_chain_spec(para_id)?)),
+		"" => Ok(Box::new(chain_spec::get_chain_spec(para_id))),
```

Comment:
Are we losing error propagation here?

Reply:
The signature of the `get_chain_spec` function changed. It no longer returns a result (if the wasm binary isn't available) it just panics. I made this change to mirror cumulus.